src/storage/src/hummock/mod.rs

```diff
@@ -106,18 +103,12 @@ impl HummockStorage {
         // TODO: separate `HummockStats` from `StateStoreMetrics`.
         stats: Arc<StateStoreMetrics>,
     ) -> HummockResult<Self> {
-        let shared_buffer_manager = Arc::new(SharedBufferManager::new(
+        LocalVersionManager::start_workers(
```

Comment:
Can we initialize the `LocalVersionManager` inside `HummockStorage` instead of passing from external? Seems only hummock uses this `LocalVersionManager`. In this way we can make sure that no one has started the worker before the local version manager is passed in.

And also make these parameter of `start_workers` of `Arc` type to be fields of the `LocalVersionManager` and make the signature to be `start_workers(self: Arc<&Self>)` and then we can simply call `local_version_manager.start_workers()`.

Reply:
> Can we initialize the LocalVersionManager inside HummockStorage instead of passing from external?

Good point. Done.

> make these parameter of start_workers of Arc type to be fields of the LocalVersionManager

It is better not to do so becuase those parameters are only used to start workers and local version manager itself doesn't need them.